TAF/testScenarios/functionalTest/V2-API/core-metadata/deviceprofile/POST-Negative-Deviceresource.robot

```diff
@@ -4,84 +4,117 @@ Resource     TAF/testCaseModules/keywords/core-metadata/coreMetadataAPI.robot
 Suite Setup  Run Keywords  Setup Suite
 ...                        AND  Run Keyword if  $SECURITY_SERVICE_NEEDED == 'true'  Get Token
 Suite Teardown  Run Teardown Keywords
-Force Tags      Skipped
-
+Force Tags      v2-api
 *** Variables ***
 ${SUITE}          Core Metadata Device Profile POST Deviceresource Negative Test Cases
+${profileName}    profileName
 
 *** Test Cases ***
 ErrProfileResourcePOST001 - Add deviceResource with Empty device profile name
     # Empty device profile name
-    When Add New Resource With Empty device profile name
-    Then Should Return Status Code "404"
+    Given Set Test Variable  ${test_profile}  Test-Profile-1
+    And Generate a device profile sample  ${test_profile}
+    And Create Device Profile ${deviceProfile}
+    And Add multiple Resources on device profile
```

Comment:
Those can be moved to `Add multiple Resources on device profile` keyword for reuse.

Reply:
updated